ports/rp2/main.c

```diff
@@ -180,7 +180,9 @@ int main(int argc, char **argv) {
     fb_alloc_init0();
     framebuffer_init0();
 
+    #if MICROPY_PY_FIR
```

Comment:
This is not defined in rp2 boards, you should first add the following to `boards/*/omv_boardconfig.cmake`:
```C
set(MICROPY_PY_FIR 1)
```